src/dotty/tools/dotc/reporting/diagnostic/messages.scala

```diff
@@ -318,4 +319,86 @@ object messages {
         |$code2
         |""".stripMargin
   }
+
+  val implicitClassRestrictionsText =
+    """For a full list of restrictions on implicit classes visit
+      |http://docs.scala-lang.org/overviews/core/implicit-classes.html""".stripMargin
```

Comment:
All interpolated strings in `hl"..."` get syntax highlighting. If you want to interpolate `implicitClassRestrictions` in you highlighted block I'd wrap it in a `NoColor(...)` `Highlight`. Already "colored" strings do not get interpolated by the interpolator.

See: https://github.com/lampepfl/dotty/blob/master/src/dotty/tools/dotc/printing/Highlighting.scala#L55